src/main/java/io/github/oliviercailloux/y2018/assisted_board_games/servlets/PlayerServlet.java

```diff
@@ -0,0 +1,77 @@
+package io.github.oliviercailloux.y2018.assisted_board_games.servlets;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+
+import com.github.bhlangonijr.chesslib.Board;
+import com.github.bhlangonijr.chesslib.Square;
+import com.github.bhlangonijr.chesslib.move.Move;
+
+import io.github.oliviercailloux.y2018.assisted_board_games.service.ChessService;
+
+/**
+ * 
+ * @author Thibaud Fremin du sartel
+ * 
+ * Ces servlets ont été conçus initialement pour communiquer avec une vue JSP.
+ * Suite à notre discussion en cours, la vue JSP a été supprimée et ces servlets seront mis à jour dans l'itération 3 
+ * pour communiquer avec les vues HTML.
+ *
+ */
+@Path("board")
+public class PlayerServlet {
+
+	private static final long serialVersionUID = 1L;
+	private static final Logger LOGGER = Logger.getLogger(PlayerServlet.class.getCanonicalName());
+
+	@PersistenceContext
+	private EntityManager em;
+
+	@Inject
+	private ChessService chessService;
+
+	@GET
+	protected void getNewBoard(@Context final HttpServletRequest request,
+			@Context final HttpServletResponse response) throws ServletException, IOException {
```

Comment:
Effacer SVP ce servlet car il est peu lisible et sera difficile à maintenir, et ne fonctionnera plus maintenant qu’il n’y a plus de JSP. Les paramètres à utiliser ne sont pas explicites dans l’en-tête.

Mieux vaut pas de fct qu’une fct pas claire.

(Vous pouvez créer un tag git pour retrouver facilement cette chose si vous voulez vous appuyer dessus, à condition de l’améliorer, lors de l’itération 3.)